onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc

```diff
@@ -255,7 +255,8 @@ TEST(ConvTransposeTest, ConvTranspose_2D_OutputShape_1) {
                         18.0f, 27.0f, 27.0f, 18.0f,
                         18.0f, 27.0f, 27.0f, 18.0f,
                         12.0f, 18.0f, 18.0f, 12.0f};
-  TestConvTransposeOp(attrs, {X, W}, {X_shape, W_shape}, expected_vals, Y_shape);
+  TestConvTransposeOp(attrs, {X, W}, {X_shape, W_shape}, expected_vals, Y_shape,
```

Comment:
see: https://github.com/microsoft/onnxruntime/blob/d12874086d6c8f6ee6a4070bcfd094b2caea7bd0/onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc#L33
unfortunately, the default seems to exclude TensorRTExecutionProvider which this overrides. 
can you add both OpenVINOExecutionProvider and TensorRTExecutionProvider for exclusion?
This is causing TensorRT CI pipeline to fail.